etna/lib/etna/clients/magma/workflows/model_synchronization_workflow.rb

```diff
@@ -161,7 +161,7 @@ def ensure_model_link(model_name, link_model_name, attribute_name)
           return if target_attributes.attribute_keys.include?(target_link_model_name)
 
           add_link = AddLinkAction.new
-          add_link.links << AddLinkDefinition.new(model_name: target_model_name, attribute_name: target_link_model_name, type: source_attribute.attribute_type)
+          add_link.links << AddLinkDefinition.new(model_name: target_model_name, attribute_name: attribute_name, type: source_attribute.attribute_type)
```

Comment:
This bug duplicated link attributes into two attributes: the intended one, and one named after the link_model.  This is intended behavior captured correctly.